packages/cyberstorm/src/components/componentStyles/PaginationButton.module.css

```diff
@@ -0,0 +1,39 @@
+.root {
+  display: flex;
+  flex: none;
+  flex-direction: row;
+  gap: var(--gap-3xs);
+  align-items: center;
+  justify-content: center;
+  min-width: 36px;
+  min-height: 36px;
```

Comment:
The notion about the min vs hard values still applies. Since both the prev/next buttons and the page numbers use the same component, you can do this in a couple of different ways; applying the appropriate css rules/class to the root or the label.

Reply:
We did talk this through and consensus was that this was ok for now iirc?